luigi/contrib/kubernetes.py

```diff
@@ -213,10 +214,15 @@ def poll_interval(self):
         """How often to poll Kubernetes for job status, in seconds."""
         return self.__DEFAULT_POLL_INTERVAL
 
+    @property
+    def pod_creation_wait_interal(self):
+        """Delay for initial pod creation for just submitted job in seconds"""
+        return self.__DEFAULT_POD_CREATION_INTERVAL
+
     def __track_job(self):
         """Poll job status while active"""
         while not self.__verify_job_has_started():
-            time.sleep(self.__DEFAULT_POLL_INTERVAL)
+            time.sleep(self.poll_interval)
```

Comment:
I think we might have steered you wrong... Or maybe i misunderstood the code the first time.

I'm looking at this more closely and have noticed that now each iteration of this `while not self.__verify_job_has_started()` we'll wait both `pod_creation_wait_interval` seconds AND `poll_interval` seconds.

This doesn't seem desirable. It also isn't clear from the user perspective - for 2 intervals to both impact the same wait time.

Am i missing something?

Reply:
I think I understand the explanation. Most likely this new added `sleep` would only happen once. Either afterwards we see pods and will always see while waiting for container running state, or we won't and assertion will be made.